src/main.c

```diff
@@ -27,14 +27,42 @@ gboolean info_noverify, info_dumpcert = FALSE;
 gboolean status_detailed = FALSE;
 gchar *output_format = NULL;
 
+#include <sys/ioctl.h>
+#include <stdio.h>
+#include <unistd.h>
```

Comment:
These should be above the other includes.

Reply:
Fixed.